utils.test.js

```diff
@@ -2398,6 +2406,11 @@ test("get licenses", () => {
       expression: "GPL-3.0-only WITH Classpath-exception-2.0",
     },
   ]);
+
+  licenses = getLicenses({
```

Comment:
This is the test for my code change. The rest of the changes are me uncommenting some other tests and fixing them so they pass.